openedx_events/event_bus/avro/tests/test_schema.py

```diff
@@ -34,6 +34,7 @@ def test_simple_schema_generation(self):
             "type": "record",
             "name": "CloudEvent",
             "doc": "Avro Event Format for CloudEvents created with openedx_events/schema",
+            'namespace': 'simple.signal',
```

Comment:
Nit: All this repetition makes me think a helper for generating this data might clean-up tests.